inst/@sym/coeffs.m

```diff
@@ -202,7 +248,7 @@
 %! assert (isequal (t, s))
 
 %!test
-%! % empty same as no specifying
+%! % empty same as not specifying
```

Comment:
SMT is probably taking the 'someweirdstuff' and making it into a sym; that is, the same as `[c, t] = coeffs(6*x*x + 27*y*x + 36, sym('s'))`.  But SMT has a legacy of accepting `char` for what should be sym inputs that we don't necessarily need to support.  I agree giving an error seems more appropriate.

That empty case is kind of annoying!  I prefer our behaviour (empty is fairly commonly the same as omitting the argument in Octave/Matlab code). I will think about this and perhaps file a separate bug.

While testing these, I noticed that the new `'all'` doesn't work with multivariate polynomials :(